input/input.go

```diff
@@ -53,8 +53,37 @@ type ScenarioParams struct {
 
 // Restraints is the restraints structure
 type Restraints struct {
```

Comment:
Small comment: In haddock2.5 (we should switch at some point) we also have support for custom symmetry restraints. But for later.